projects/core/src/routing/configurable-routes/config/default-storefront-routes-translations.ts

```diff
@@ -0,0 +1,55 @@
+import { StorefrontRoutesTranslations } from './storefront-routes-translations';
+
+export const defaultStorefrontRoutesTranslations: {
+  default?: StorefrontRoutesTranslations;
+  [languageCode: string]: StorefrontRoutesTranslations;
+} = {
+  default: {
+    homepage: { paths: [''] },
+    cart: { paths: ['cart'] },
+    search: { paths: ['search/:query'] },
+    login: { paths: ['login'] },
+    register: { paths: ['register'] },
+    resetNewPassword: { paths: ['reset-new-password/:token'] },
```

Comment:
forgotPassword

Reply:
I renamed config key names: `resetPassword` to `forgotPassword` nad `resetNewPassword` to `resetPassword`.
However, I left paths untouched.